test/functionalTest/cases/3707_get_or_change_log_config/get_or_change_log_config.test

```diff
@@ -0,0 +1,488 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Get or Change Log configs via REST
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET initial log config 
+# 02. PUT /admin/log and error
+# 03. GET log config, see initial log config
+# 04. PUT /admin/log?level=ERROR and see it work
+# 05. GET log config, see level=ERROR
+# 06. PUT /admin/log?level=xyz and see error
+# 07. GET log config, see previous data
+# 08. PUT /admin/log?level=INFO and see it work
+# 09. GET log config, see level=INFO
+# 10. PUT /admin/log?infoPayloadMaxSize=5000 and see it work
+# 11. GET log config, see infoPayloadMaxSize=5000
+# 12. PUT /admin/log?infoPayloadMaxSize=100000 and see it work
+# 13. GET log config, see infoPayloadMaxSize=100000
+# 14. PUT /admin/log?infoPayloadMaxSize=-2000 and see error
```

Comment:
I'd suggest to add a couple of cases with non integer number. For instance:

```
xx. PUT /admin/log?infoPayloadMaxSize=3.1416 and see error
xx. PUT /admin/log?infoPayloadMaxSize=nonumer and see error
```

Reply:
Fixed. Added cases in [24de8d9](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/24de8d9e544ab1ddba1af46cdb70ded49bb85e2e)